integration-tests/src/test/java/io/nem/symbol/sdk/infrastructure/MerkleLoaderIntegrationTest.java

```diff
@@ -66,110 +61,164 @@ public class MerkleLoaderIntegrationTest extends BaseIntegrationTest {
 
   public static final OrderBy ORDER_BY = OrderBy.DESC;
 
-  public List<Arguments> mosaicRestriction() {
+  @ParameterizedTest
+  @EnumSource(RepositoryType.class)
+  void mosaicRestrictionMerkles(RepositoryType repositoryType) {
+    mosaicRestriction()
+        .forEach(
+            mosaicRestriction -> {
+              StateProofServiceImpl service =
+                  new StateProofServiceImpl(getRepositoryFactory(repositoryType));
+              StateMerkleProof<MosaicRestriction<?>> proof =
+                  get(service.mosaicRestriction(mosaicRestriction));
```

Comment:
O, and shouldn't be changed within this change:
noticed it earlier as well, what's your line-wrap settings? (there are statements that don't look like super long lines and still get wrapped)

Reply:
I'm using formatter settings originally defined by spotless plugin:
- https://github.com/diffplug/spotless/tree/main/plugin-gradle#java
- https://github.com/symbol/sdk-java/blob/dev/build.gradle#L105

I'm doing formatting just by `./gradlew spotlessApply`
As I can see max line length is defined as 100 which is IMO too low too but changing that will reformat the entire project which I want to avoid :)